redaxo/src/addons/phpmailer/lib/mailer.php

```diff
@@ -56,19 +56,44 @@ public function send()
             if ($this->log) {
                 $this->log();
             }
-            return parent::send();
+            $addon = rex_addon::get('phpmailer');
+            if(!parent::send() && $addon->getConfig('logging') != 0){
+            $this->toMailerLog('ERROR');
+            return false;
+            }
+       
+        if($addon->getConfig('logging') == 2)
+        {
+        $this->toMailerLog('OK');
+        }
+        return true;
         });
```

Comment:
In den 10 Zeilen hier drüber passt einiges mit der Einrückung nicht.
(Teils auch noch an anderen Stellen)

Reply:
sollte jetzt passen

Reply:
https://github.com/redaxo/redaxo/pull/3109/commits/a7892fd13e212bd65cafe2040324f78d17e0a4c9